src/test/java/net/snowflake/ingest/TestUtils.java

```diff
@@ -62,18 +79,20 @@
   private static void init() throws Exception {
     profile = (ObjectNode) mapper.readTree(new String(Files.readAllBytes(Paths.get(PROFILE_PATH))));
 
-    user = profile.get("user").asText();
-    account = profile.get("account").asText();
-    port = profile.get("port").asInt();
-    ssl = profile.get("ssl").asText();
-    database = profile.get("database").asText();
-    connectString = profile.get("connect_string").asText();
-    schema = profile.get("schema").asText();
-    warehouse = profile.get("warehouse").asText();
-    host = profile.get("host").asText();
-    scheme = profile.get("scheme").asText();
+    user = profile.get(USER).asText();
+    account = profile.get(ACCOUNT).asText();
+    port = profile.get(PORT).asInt();
+    ssl = profile.get(SSL).asText();
+    database = profile.get(DATABASE).asText();
+    connectString = profile.get(CONNECT_STRING).asText();
+    schema = profile.get(SCHEMA).asText();
+    warehouse = profile.get(WAREHOUSE).asText();
+    host = profile.get(HOST).asText();
+    scheme = profile.get(SCHEME).asText();
+
+    role = Optional.ofNullable(profile.get(ROLE)).map(r -> r.asText()).orElse("DEFAULT_ROLE");
```

Comment:
ah, could we do this in the SDK logic instead of in the test, seems like a good logic to have? (if I understand correctly, we will use the DEFAULT_ROLE is there is no role input?)